test/unit/access-point.js

```diff
@@ -294,3 +294,121 @@ exports['Tessel.prototype.disableAccessPoint'] = {
       });
   }
 };
+
+exports['Tessel.prototype.disableAccessPoint'] = {
+  setUp: function(done) {
+    this.sandbox = sinon.sandbox.create();
+    this.getAccessPointInfo = this.sandbox.spy(Tessel.prototype, 'getAccessPointInfo');
+    this.getAccessPointConfig = this.sandbox.spy(commands, 'getAccessPointConfig');
+    this.getAccessPointIP = this.sandbox.spy(commands, 'getAccessPointIP');
+
+    this.tessel = TesselSimulator();
+
+    done();
+  },
+
+  tearDown: function(done) {
+    this.tessel.mockClose();
+    this.sandbox.restore();
+    done();
+  },
+
+  returnsActiveInfo: function(test) {
+    test.expect(3);
+    var results = {
+      ssid: 'TestSSID',
+      password: 'TestPass123',
+      security: 'psk2',
+      disabled: '0',
+      ip: '192.168.200.1'
+    };
+
+    // Test is expecting two closes...;
+    this.tessel._rps.on('control', (command) => {
+      if (command.toString() === 'uci show wireless.@wifi-iface[1]') {
+        var info = new Buffer(
+          `wireless.cfg053579.ssid='${results.ssid}'
+          wireless.cfg053579.key='${results.password}'
+          wireless.cfg053579.encryption='${results.security}'
+          wireless.cfg053579.disabled='0'`
+        );
```

Comment:
Just a heads up, this will preserve all the indentation white space:

```js   
var buffer = new Buffer(
  `a
   b
   c`
);

console.log(buffer);
console.log(buffer.length);
``` 

Result: 

```
a
   b
   c
11
``` 

Take a look in deploy.js, where I use the `tags.stripIndent` template tag from the "common-tags" module:

- https://github.com/tessel/t2-cli/blob/master/lib/tessel/deploy.js#L10
- https://github.com/tessel/t2-cli/blob/master/lib/tessel/deploy.js#L971-L974